icons/README.md

```diff
@@ -2,6 +2,8 @@
 
 These icons are a way to standardize Kubernetes architecture diagrams for presentation. Having uniform architecture diagrams improve understandibility.
 
+Icon for Kubernetes it self can find at https://github.com/kubernetes/kubernetes/tree/master/logo .
```

Comment:
```suggestion
The icon for Kubernetes itself can be found at https://github.com/kubernetes/kubernetes/tree/master/logo .
```
+1 to add this.

Reply:
fixed https://github.com/kubernetes/community/pull/7921/commits/e0841081d244822c82dc0d46871030f86ec53311